pkgs/tools/misc/x11idle/default.nix

```
@@ -1,12 +1,13 @@
-{ stdenv, fetchgit, libXScrnSaver, libX11 }:
+{ stdenv, xlibs, fetchurl, libXScrnSaver, libX11 }:
```

remove unneeded `xlibs`:

```suggestion
{ stdenv, fetchurl, libXScrnSaver, libX11 }:
```

this should also make the build pass.